content/blog/2017/2017-02-10-declarative-html-publisher.adoc

```diff
@@ -0,0 +1,322 @@
+---
+layout: post
+title: "Declarative Pipeline: Publishing HTML Reports"
+tags:
+- tutorial
+- pipeline
+- declarative
+- plugins
+- ruby
+author: lnewman
+---
+
+NOTE: This is a guest post by link:https://github.com/bitwiseman[Liam Newman],
+Technical Evangelist at link:https://cloudbees.com[CloudBees].
+
+**Declare Your Pipelines!**
+link:/blog/2017/02/03/declarative-pipeline-ga/[Declarative Pipeline 1.0 is here]!
+This is the second post in a series showing some of the cool features of
+link:/doc/book/pipeline/syntax/#declarative-pipeline[Declarative Pipeline].
+
+In the
+link:/blog/2017/02/07/declarative-maven-project/[previous blog],
+we created a simple Declarative Pipeline.
+For this blog, we'll go back and look at the Scripted Pipeline for the
+link:/blog/2016/07/01/html-publisher-plugin/[Publishing HTML Reports in Pipeline] blog.
```

Comment:
@orrc - the problem is I'm talking about a `post` block in this post.  
I'm going to be posting a post about `post`.  😢